templates/k8s-ansible-service-broker.yaml.j2

```diff
@@ -29,7 +29,7 @@ rules:
   verbs: ["create", "delete"]
 - apiGroups: ["automationbroker.io"]
   attributeRestrictions: null
-  resources: ["bundles", "bundlebindings", "bundleinstances"]
+  resources: ["bundles", "bundlebindings", "bundleinstances", "jobstates"]
```

Comment:
If your using bundle binding/bundle instances then job states should not be installed in the cluster. I think we should remove this IMO.

Reply:
agreed let's remove the jobstates.